### ISSUE TITLE
Material button colors don't fit M3 design scheme and a11y

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/Uno.Themes#934]
## Current behavior

The current material foreground/background colors fail all accessibility checks

<img width="72" alt="image" src="https://user-images.githubusercontent.com/1075116/220882921-73ade472-36dd-40e3-bd2d-20851ecd4f39.png">
<img width="81" alt="image" src="https://user-images.githubusercontent.com/1075116/220882980-67a9ebba-e6cd-4e2e-b96e-be1ea4e0b229.png">

## Expected behavior

https://m3.material.io/components/buttons/specs

<img width="435" alt="image" src="https://user-images.githubusercontent.com/1075116/220882794-2a1b732d-ff49-4a0b-952f-6d5ed1c23e62.png">

## Environment

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] UWP
- [ ] MacOS

**Comment by MartinZikmund** [comment]
@NVLudwig

**Comment by NVLudwig** [comment]
The colors we use are the Uno brand colors, they look a lot like the default Material 3 by coincidence. We have a separate issue logged for the Dark mode colors. I'll post here once the fix is applied every where
